docs/Pipelines/PET_Linear.md

```diff
@@ -0,0 +1,111 @@
+resulting in a standardized uptake value ratio (SUVR) map;
+- cropping of the registered images to remove the background.
+
+## Prerequisites
+
+You need to have performed the [`t1-linear`](../T1_Linear) pipeline on your
+T1-weighted MR images.
+
+## Dependencies
+
+If you only installed the core of Clinica, this pipeline needs the installation
+of **ANTs** on your computer. You can find how to install this software package
+on the [third-party](../../Third-party) page.
+
+## Running the pipeline
+
+The pipeline can be run with the following command line:
+
+```text
+clinica run pet-linear <bids_directory> <caps_directory> <acq_label> <suvr_reference_region>
+```
+
+where:
+
+- `bids_directory` is the input folder containing the dataset in a
+[BIDS](../../BIDS) hierarchy;
+- `caps_directory` is the output folder containing the results in a
+[CAPS](../../CAPS/Introduction) hierarchy;
+- `acq_label` is the label given to the PET acquisition, specifying the tracer
+used (`acq-<acq_label>`). It can be for instance 'fdg' for
+<sup>18</sup>F-fluorodeoxyglucose or 'av45' for <sup>18</sup>F-florbetapir;
+- `suvr_reference_region` is the reference region used to perform intensity
+normalization (i.e. dividing each voxel of the image by the average uptake in
+this region) resulting in a standardized uptake value ratio (SUVR) map. It can
+be `cerebellumPons` (used for amyloid tracers) or `pons` (used for FDG).
+
+By default, cropped images (matrix size 169×208×179, 1 mm isotropic voxels) are
+generated to reduce the computing power required when training deep learning
+models. Use the `--uncropped_image` flag if you do not want to crop the image.
+
+The pipeline also offers the possibility to save the PET image in the T1w space
+after rigid transformation using the `--save_pet_in_t1w_space` flag.
+
+!!! note
+    The arguments common to all Clinica pipelines are described in
+    [Interacting with Clinica](../../InteractingWithClinica).
+
+## Outputs
+
+Results are stored in the following folder of the [CAPS hierarchy](../../CAPS/Specifications/#pet-imaging-data):
+`subjects/<participant_id>/<session_id>/pet_linear`.
+
+The main output files are:
+
+- `<source_file>_space-MNI152NLin2009cSym_desc-Crop_res-1x1x1_suvr-<label>_pet.nii.gz`:
+PET SUVR image registered to the [`MNI152NLin2009cSym` template](https://bids-specification.readthedocs.io/en/stable/99-appendices/08-coordinate-systems.html)
+and cropped.
+- `<source_file>_space-T1w_rigid.mat`: rigid transformation between the PET and
+T1w images estimated with [ANTs](https://stnava.github.io/ANTs/).
+- (optional) `<source_file>_space-MNI152NLin2009cSym_res-1x1x1_pet.nii.gz`:
+PET SUVR image affinely registered to the [`MNI152NLin2009cSym` template](https://bids-specification.readthedocs.io/en/stable/99-appendices/08-coordinate-systems.html)
+(i.e. not cropped).
+- (optional) `<source_file>_space-T1w_pet.nii.gz`: PET image affinely registered
+to the associated T1w image.
+
+## Going further
```

Comment:
I don't think that this paragraph is currently true. It may need to be commented out for now.

Reply:
After discussion with Elina, it seems that it may work with correct options so we will keep this paragraph. However it can be improved in deep-learning prepare data pipeline.